.travis.yml

```diff
@@ -1,10 +1,26 @@
 language: ruby
-rvm:
-  - 2.1
-  - 2.2
 before_install:
   - gem update bundler
-gemfile:
-  - gemfiles/fluentd_0_10.gemfile
-  - gemfiles/fluentd_0_12.gemfile
-  - gemfiles/fluentd_0_14.gemfile
+
+matrix:
+  include:
+    - rvm: 2.1
+      gemfile: gemfiles/fluentd_0_10.gemfile
```

Comment:
Hmm, I don't want to migrate out_map to use v0.14.
Because out_map can replace with filter_map and label feature.